server/index.js

```diff
@@ -1,21 +1,23 @@
 import express from 'express';
 import swaggerUi from 'swagger-ui-express';
 import apiDocumentation from '../swagger.json';
+import importAuthUserRoute from './routes/authUserRoute';
 
 const app = express();
+const basePath = '/api/v-unknown';
```

Comment:
I think '/api' would suffice.  '/api' here is enough

Reply:
Ooohhkk... I  was guessing maybe we will add a version of our application that why I added Version Unknown but it fine.